### PR TITLE
Fix bug with multiple imports, part of issue #706

### PR DESCRIPTION
**Pull request by liscju**
Problem was checking for pre previous line with imports instead of checking for two previous lines. As suggested in previous PR i introduce method hasNotAllowedFewLinesBeforeEmpty which checks if last two lines is empty and multiple empty lines are empty. I changed every checking "!allowMultipleEmptyLines && isPrePreviousLineEmpty(ast)" to hasNotAllowedFewLinesBeforeEmpty. In previous PR @romani suggested putting more light hasEmptyLineBefore function before isPrePreviousLineEmpty, i did not do this because hasEmptyLineBefore function does not check if previous line exist. I certainly can put that guard in hasEmptyLineBefore and put it before isPrePreviousLineEmpty in condition, but in this situation hasEmptyLineBefore would not be any lighter than isPrePreviousLineEmpty but im open to further suggestions.

### CODE REVIEW

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java [hunk 1]
```diff
@@ -279,6 +278,17 @@ else if (!allowNoEmptyLineBetweenFields
     }
 
     /**
+     * Checks if a token has empty two previous lines and there is not allowed multiple empty lines
+     * @param token DetailAST token
+     * @return true, if token has empty two lines before and allowMultipleEmptyLines is false
+     */
+    private boolean hasNotAllowedFewLinesBeforeEmpty(DetailAST token)
```
Comment (mkordas) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java]: I think the better name would be `hasNotAllowedTwoEmptyLinesBefore()`

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java [hunk 2]
```diff
@@ -279,6 +278,17 @@ else if (!allowNoEmptyLineBetweenFields
     }
 
     /**
+     * Checks if a token has empty two previous lines and there is not allowed multiple empty lines
```
Comment (mkordas) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyLineSeparatorCheck.java]: Change `there is not allowed multiple empty lines` to `multiple empty lines are not allowed`

**Comment by liscju** [comment]
Changed according to @mkordas suggestions

**Comment by romani** [comment]
Please swap hasEmptyLineBefore and isPrePreviousLineEmpty at method hasNotAllowedTwoEmptyLinesBefore to make it a little bit more logical to test nearest line before

**Comment by liscju** [comment]
I swapped this condition as requested and added needed in this situation guard to hasEmptyLineBefore

**Comment by romani** [comment]
merged as FF, thanks a lot !!!!

**Pull request closed without merging**